### ISSUE TITLE
[Feature Request] : Modern editor experience for meesages

### ISSUE THREAD

**Issue by Xyncgas** [telegramdesktop/tdesktop#26145]
### Is your feature request related to a problem?

It's a proposal for a telegram feature

### Describe the solution you'd like

Telegram already displays a great capability for displaying messages, if we can create / modify a message in a prestige way so other UI components such as list of channels or other people's conversation / other messages are hidden away, the application become a text editor that a journalist is willing to write an entire report as if it's an actual editor, and the report may be sent to a group / channel by the author, while a bot can pick it up and publish it to websites or twitter etc

### Describe alternatives you've considered

Instead of creating the editor experience directly inside telegram, I can just create another application based on telegram by myself, or independently without using telegram bot.

**Comment by VetaLioSTV** [comment]
forwarding multiple message from chat for now works as flat list which is ultra-ultra bad, idk... but should be reworked as feature request also..

**Comment by what1s1ove** [comment]
> Rather than creating an editor within Telegram, which would add unnecessary bloat to the app, it would be better if Telegram implemented Markdown support. This way, you could use a standalone editor with Markdown capabilities and then easily copy and paste the formatted text into Telegram.

Agreed. While Markdown is supported to some extent, many constructs are still missing. In my opinion, it's actually less convenient because when you write one part of Markdown (for example, a code block using three backticks), you expect the other part (like quoting with the `>` symbol) to work as well. Full support is indeed lacking.
